<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>随机数</string>
           </property>
           <widget class="QListWidget" name="randomList">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>131</width>
              <height>451</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>冒泡排序</string>
           </property>
           <widget class="QListWidget" name="bubbleList">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>131</width>
              <height>451</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>快速排序</string>
           </property>
           <widget class="QListWidget" name="quickList">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>131</width>
              <height>451</height>
             </rect>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
